packages/LUIS/bin/luis.js

```diff
@@ -226,6 +226,9 @@ async function runProgram() {
                 case "sublist":
                     result = await client.model.addSubList(args.region, args.cloud, args.appId, args.versionId, args.clEntityId, requestBody, args);
                     break;
+                case "appazureaccount":
+                    result = await client.azureAccounts.assignToApp(args.region, args.cloud, args.appId, requestBody, args);
```

Comment:
when testing, this gave an incorrectly formatted authorization header. Will fix it before merging.

Reply:
resolved